protocol/v2/qbft/instance/round_change.go

```diff
@@ -44,6 +44,8 @@ func (i *Instance) uponRoundChange(
 		zap.Uint64("msg_round", uint64(msg.QBFTMessage.Round)),
 	)
 
+	i.metrics.RecordRoundChange(ctx, msg.QBFTMessage.Round, receivedReason)
```

Comment:
Do we need to record it? We do it once again below depending on the if branch:

```go
i.metrics.RecordRoundChange(ctx, msg.QBFTMessage.Round, justifiedReason)
```
```go
i.metrics.RecordRoundChange(ctx, newRound, partialQuorumReason)
```

I think we don't need `receivedReason`, do we?

Reply:
you are right, `reveivedReason`does seem redundant since im already recording more specific reasons, fixed